src/explore-education-statistics-frontend/src/modules/data-catalogue/components/DownloadStep.tsx

```diff
@@ -0,0 +1,159 @@
+}: Props & InjectedWizardProps) => {
+  const { isActive, currentStep, stepNumber } = stepProps;
+
+  const stepEnabled = currentStep > stepNumber;
+  const stepHeading = (
+    <WizardStepHeading {...stepProps} fieldsetHeading stepEnabled={stepEnabled}>
+      <span className="dfe-flex dfe-align-items--center">
+        Choose files to download{' '}
+        {isLatestRelease && (
+          <Tag strong className="govuk-!-margin-left-4">
+            This is the latest data
+          </Tag>
+        )}
+        {!isLatestRelease && (
+          <Tag strong colour="orange" className="govuk-!-margin-left-4">
+            This is not the latest data
+          </Tag>
+        )}
+      </span>
+    </WizardStepHeading>
+  );
+
+  const getTimePeriod = (subject: Subject) => {
+    const { from, to } = subject.timePeriods;
+
+    if (from && to) {
+      return from === to ? from : `${from} to ${to}`;
+    }
+
+    return from || to;
+  };
+
+  const checkboxOptions = useMemo<CheckboxOption[]>(
+    () =>
+      subjects.map(subject => {
+        const { content } = subject;
+        const geographicLevels = [...subject.geographicLevels]
+          .sort()
+          .join('; ');
+        const timePeriod = getTimePeriod(subject);
+        const hasDetails = content || geographicLevels || timePeriod;
+
+        return {
+          label: `${subject.name} (${subject.downloadFile.extension}, ${subject.downloadFile.size})`,
+          value: subject.downloadFile.id,
+          hint: hasDetails ? (
+            <Details summary="More details" className="govuk-!-margin-bottom-2">
+              <h4>This subject includes the following data:</h4>
+              <SummaryList>
+                {geographicLevels && (
+                  <SummaryListItem term="Geographic levels">
+                    {geographicLevels}
+                  </SummaryListItem>
+                )}
+
+                {timePeriod && (
+                  <SummaryListItem term="Time period">
+                    {timePeriod}
+                  </SummaryListItem>
+                )}
+
+                {content && (
+                  <SummaryListItem term="Content">
+                    <ContentHtml html={content} />
+                  </SummaryListItem>
+                )}
+              </SummaryList>
+            </Details>
+          ) : null,
+        };
+      }),
+    [subjects],
+  );
+
+  return (
+    <Formik<DownloadFormValues>
+      enableReinitialize
+      initialValues={initialValues}
+      validateOnBlur={false}
+      validationSchema={Yup.object<DownloadFormValues>({
+        files: Yup.array().of(Yup.string()).required('Choose a file'),
+      })}
+      onSubmit={async ({ files }) => {
+        await onSubmit({ files });
+      }}
+    >
+      {form => {
+        return isActive ? (
+          <Form {...form} id="downloadForm" showSubmitError>
+            {checkboxOptions.length > 0 ? (
+              <>
+                <FormFieldCheckboxGroup<DownloadFormValues>
```

Comment:
Personal opinion, but the alignment on the checkboxes and the label/hint looks a bit off? 

Maybe they might look a bit nicer if the text is aligned with the top of the checkbox rather than the middle in this case?

Reply:
I get what you mean, but they look a bit crowded together it top align the text. Tried with small checkboxes as well but that looks worse. Might be one for @mrmarvino to have a quick look at.